improve single cell

stemangiola opened the pull request.

stemangiola committed:
improve single cell

stemangiola committed:
improve single cell

stemangiola committed:
improve introduction

stemangiola committed:
improve introduction

mblue9 commented on vignettes/tidytranscriptomics.Rmd:
```diff
@@ -941,6 +941,10 @@ Questions:
 
 # Part 3 Single-cell RNA-seq
 
+In the Part 2 we mentioned the importance of studying the cell-type composition of a biological sample (i.e. bulk RNA sequencing deconvolution). The barcoding of RNA molecules according to the single cell origin gives higher information resolution. One of the benefits is a more direct estimation of cell-type composition. While for bulk RNA sequencing we need to use inference, with single-cell RNA ...
```
```suggestion
In Part 2 we showed how we can study the cell-type composition of a biological sample using bulk RNA sequencing.  Single cell sequencing enables a more direct estimation of cell-type composition and gives greater resolution. For bulk RNA sequencing we need to infer the cell types using the abundance of transcripts in the whole sample, with single-cell RNA sequencing we can directly measure the transcripts in each cell and then classify the cells into cell types. 
```

mblue9 replied:
What about something like this? I think we should not mention technical details like barcoding unless we're going to explain it.

mblue9 commented on vignettes/tidytranscriptomics.Rmd:
```diff
@@ -941,6 +941,10 @@ Questions:
 
+In the Part 2 we mentioned the importance of studying the cell-type composition of a biological sample (i.e. bulk RNA sequencing deconvolution). The barcoding of RNA molecules according to the single cell origin gives higher information resolution. One of the benefits is a more direct estimation of cell-type composition. While for bulk RNA sequencing we need to use inference, with single-cell RNA ...
+
+Seurat is a very popular analysis framework for single cell RNA sequencing data [@butler2018integrating; @stuart2019comprehensive] .
```
```suggestion
Seurat is a very popular analysis toolkit for single cell RNA sequencing data [@butler2018integrating; @stuart2019comprehensive] .
```

mblue9 commented on vignettes/tidytranscriptomics.Rmd:
````diff
@@ -941,13 +941,28 @@ Questions:
 
 
 ```{r message=FALSE, warning=FALSE}
+# Detach all previous packages - stackoverflow.com/questions/7505547/detach-all-packages-while-working-in-r
````
It's a bit of a shame to have to add this base R lump after showing all the nice tidyverse code. An alternative could be to run Session > Restart R to clear everything that's been loaded but if you want to stick with this that's fine.

stemangiola replied:
The point of this is because one generally either do bulk analyses or single cell. And so we can keep avoiding ``tidyseurat::xxx``

The problem to restart is that is not programmatic and the automatic vignette compilation could fail (i've took off all ``tidyseurat::xxx``). Maybe we could find a more elegant alternative

mblue9 replied:
Oh yes forgot about the need to auto build. Could you add a Note to mention you wouldn't usually need to do tidyseurat::xxx?  As just realised if you detach all packages here the sessionInfo at the end under "Reproducibility" will now no longer reflect what was used it'll only show the packages from the single-cell part.

mblue9 committed:
Update project access instructions

stemangiola committed:
Update vignettes/tidytranscriptomics.Rmd

Co-authored-by: mblue9 <maria.doyle@petermac.org>

stemangiola committed:
Update vignettes/tidytranscriptomics.Rmd

Co-authored-by: mblue9 <maria.doyle@petermac.org>

stemangiola committed:
Update vignettes/tidytranscriptomics.Rmd

Co-authored-by: mblue9 <maria.doyle@petermac.org>

stemangiola committed:
Update vignettes/tidytranscriptomics.Rmd

Co-authored-by: mblue9 <maria.doyle@petermac.org>

mblue9 committed:
Add link to more info on normalisation

mblue9 committed:
Add slide on types of transcriptomic analyses

mblue9 committed:
Small edits

mblue9 committed:
Merge pull request #31 from stemangiola/edits9

Edits9

mblue9 committed:
Update background.Rmd

mblue9 committed:
Update background.Rmd

mblue9 committed:
Add link to the more detailed schedule

mblue9 committed:
Small edits

mblue9 committed:
Merge pull request #32 from stemangiola/edits10

Edits10

mblue9 committed:
Remove details on gate coordinates

mblue9 committed:
Add link to background webpage in main Rmd

mblue9 committed:
Merge pull request #33 from stemangiola/edits11

Edits11

stemangiola committed:
keep libraries

stemangiola committed:
keep libraries

mblue9 commented on vignettes/tidytranscriptomics.Rmd:
```diff
@@ -973,13 +981,29 @@ pbmc_small_tidy
 
+## Polish the data
+
+We can interact with out object as we do with any tibble. In this case we want to polish some annotation column.
```
```suggestion
We can interact with our object as we do with any tibble. In this case we want to polish an annotation column.
```

stemangiola committed:
keep libraries

stemangiola committed:
keep libraries

mblue9 commented on vignettes/tidytranscriptomics.Rmd:
```diff
@@ -1064,14 +1090,20 @@ blueprint <- celldex::BlueprintEncodeData()
 # Infer cell identities
 cell_type_df <-
+	
+	# extracting counts from Seurat object
```
```suggestion
# extracting counts from Seurat object
```

mblue9 replied:
I think the comments are not in line with the code in this block?

mblue9 commented:
Can you rebase this PR against master? as I think the old content in this PR is overwriting the changes I've made to remove the attr(gate) here https://github.com/stemangiola/rpharma2020_tidytranscriptomics/pull/33/files 
it's still showing up in the website here https://stemangiola.github.io/rpharma2020_tidytranscriptomics/articles/tidytranscriptomics.html

stemangiola committed:
re-flow

stemangiola committed:
re-flow

stemangiola committed:
resolve conflicts

Merge branch 'update-single-cell' of https://github.com/stemangiola/rpharma2020_tidytranscriptomics into update-single-cell

Conflicts:
	vignettes/tidytranscriptomics.Rmd

stemangiola commented:
> Can you rebase this PR against master? as I think the old content in this PR is overwriting the changes I've made to remove the attr(gate) here https://github.com/stemangiola/rpharma2020_tidytranscriptomics/pull/33/files
> it's still showing up in the website here https://stemangiola.github.io/rpharma2020_tidytranscriptomics/articles/tidytranscriptomics.html

rebased

mblue9 commented:
I think something went wrong here? as you've added old commits into this PR, see here for how to fix https://stackoverflow.com/questions/50780902/accidentally-added-others-commits-to-my-pr-using-git-rebase

stemangiola committed:
Update vignettes/tidytranscriptomics.Rmd

Co-authored-by: mblue9 <maria.doyle@petermac.org>

stemangiola committed:
Update vignettes/tidytranscriptomics.Rmd

Co-authored-by: mblue9 <maria.doyle@petermac.org>

stemangiola commented:
> mits into this PR, see her

I am not sure what you are referring to. Could you be more specific? Maybe I can resolve manually some inconsistencies

stemangiola committed:
clean up

stemangiola commented:
@mblue9 I would like to merge this pull request anyway because the error is about a code (celldex) that is not present in any file anymore. I would like to see if it gets resolved. What do you think?

mblue9 commented:
Ok that’s so weird ..more weirdness here. I’ll merge it as I want to merge mine then.
